src/main/java/space/obminyashka/items_exchange/service/impl/SendGridService.java

```diff
@@ -63,17 +64,18 @@ private void throwNotFoundException() throws EmailValidationCodeNotFoundExceptio
         throw new EmailValidationCodeNotFoundException(getMessageSource(ResponseMessagesHandler.ExceptionMessage.EMAIL_NOT_FOUND_OR_EXPIRED));
     }
 
-    private Request createMailRequest(Mail mail) throws IOException {
+    private Request createMailRequest(Mail mail, UUID codeId) throws IOException {
+        var apiPrefix = EMAIL_VALIDATE_CODE.substring(0, 23);
```

Comment:
Looks like magic numbers. 
You can make it a way better by simply using String.replace method

Reply:
Done